src/commands/build.ts

```diff
@@ -1,17 +1,29 @@
 import * as fs from 'fs';
+import {GlobalOptions} from './global';
 import {Pod} from '../pod';
 
 interface BuildOptions {
   outputDirectory?: string;
 }
 
 export class BuildCommand {
-  constructor(private readonly options: BuildOptions) {
+  private readonly globalOptions: GlobalOptions;
+  private readonly options: BuildOptions;
+
+  constructor(globalOptions: GlobalOptions, options: BuildOptions) {
+    this.globalOptions = globalOptions;
     this.options = options;
   }
 
   async run(path = './') {
     const pod = new Pod(fs.realpathSync(path));
-    await pod.builder.export();
+    const command = pod.profiler
```

Comment:
Rather than making the profiler an option can it just be "always on" like the translation recording? Other static generators (i.e. 11ty) warn you when one metric starts going too slowly. If we can make these things always on without introducing a real performance hit then I think we should do it.

Reply:
The profiler is always on currently. The if statement a couple of lines later controls when the profiler shows a report of the results to the console. I didn't think you would always want the profiling to show.

It would be good to figure out which metrics are going slowly, but we don't have baselines for them right now. So in the future we could make it only show when there are performance degradations.

For now I have it to use the `--profile` global flag for showing the report numbers. Unless you want it to always show them?

Reply:
Good that it's always on. :) And yeah; while we're early in the project let's always show them. I think that will be helpful.

- After we see how things are going, we can determine thresholds for actually displaying them (i.e. if any single metric takes up more than X% of time, we show that offending metric).
- After we conditionally show metrics based on a threshold, we can make a `--profile` flag turn on/off the display for people that just want to see all metrics.

All this is along the lines of "sane defaults" and making sure that the defaults communicate useful information. :D (I bet we were the only users of the `--profile` arg and tools with Grow and probably few/no other users gave it a try...)

Reply:
I've updated it show it always shows the report and lists out any timers that tool over `20%` of the total time currently. We can adjust specific timers to have different thresholds. And the `--profile` flag shows an expanded set of timers.

Reply:
Can we drop `wrapAsync` here too and just go with the `try ... finally` approach? I like the consistency + how easy it is to understand and doesn't mess with how the code is written.